zabbix/packaging/postinst

```diff
@@ -1,13 +1,20 @@
 #!/bin/bash
-
+  
 CONFFILE="/opt/rudder/etc/zabbix.conf"
+CRONFILE="/etc/cron.d/zabbix-rudder"
 
 mkdir -p /var/rudder/plugin-resources
-echo "0 * * * * /opt/rudder/bin/zabbix-plugin.py update " > /etc/cron.d/zabbix-rudder
+
+# create cron file
+if [ ! -f "${CRONFILE}"]; then
```

Comment:
we should probably replace it, I don't know why this test. Do you know @peckpeck?

Reply:
it was to avoid changing user modified cron, in some case it is probably better to replace it